v2/pkg/output/output.go

```diff
@@ -57,7 +61,7 @@ type ResultEvent struct {
 	// TemplatePath is the path of template
 	TemplatePath string `json:"-"`
 	// Info contains information block of the template for the result.
-	Info map[string]interface{} `json:"info,inline"`
+	Info model.Info `json:"info,inline"`
```

Comment:
The problem with this is that the generated json is quite unreadable and not really parsable and consistent with the rest of the son. 

```
{"templateID":"cname-service-detector","info":{"Name":"3rd party service checker","Authors":{"Value":["","pdteam"]},"Tags":{"Value":["","dns"]},"Description":"","Reference":{"Value":null},"SeverityHolder":{"Severity":0}},"matcher_name":"github","type":"dns","host":"docs.hackerone.com","matched":"docs.hackerone.com","timestamp":"2021-08-11T13:00:07.840694+05:30"}
```

We need to change the behaviour somehow to make the output consistent in case of json.

Reply:
Done in f56ff199d60ae627e57124521e748e24ca0e439e